core/templates/pages/community-dashboard-page/translation-language-selector/translation-language-selector.component.spec.ts

```diff
@@ -0,0 +1,162 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TranslationLanguageSelectorComponent } from
+  // eslint-disable-next-line max-len
+  'pages/community-dashboard-page/translation-language-selector/translation-language-selector.component.ts';
+import { ContributionOpportunitiesBackendApiService } from
+  // eslint-disable-next-line max-len
+  'pages/community-dashboard-page/services/contribution-opportunities-backend-api.service.ts';
+import { FeaturedTranslationLanguageObjectFactory } from
+  'domain/opportunity/FeaturedTranslationLanguageObjectFactory';
```

Comment:
Change all these imports to be after fileoverview line.

Reply:
Done